Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM registry.suse.com/bci/bci-base:15.4
+
+LABEL org.opencontainers.image.source="https://github.com/trento-project/checks"
+
+# If set to C, LC_ALL takes precedence
+ENV LC_ALL=C.UTF-8
+
+# tar is required by kubectl cp
+RUN zypper --non-interactive in -y tar && \
+    zypper --non-interactive clean
+
```

Comment:
nitpick: (odc). I would remove the double new lines that are across the file. They make it the file look "not cared"

Reply:
You are actually right

Reply:
Thank you! It looks more professional (to me at least XD)